devel/go-tools/Portfile

```diff
@@ -14,11 +14,11 @@ maintainers         {ciserlohn @ci42}
 description         Various packages and tools that support the Go programming language.
 long_description    $description
 
-checksums           rmd160  97410382ccbe3efce08e96756c63db33eb8b1703 \
-                    sha256  c451b5af8a10b06624d5ae92c33aa1b30beaf380e195d1373b333fde1553c15a \
-                    size    2484284
-		    
-depends_build	    port:go
+checksums           rmd160  861f7f72627b4449752dded9ab7c1a376435e841 \
+                    sha256  b26d3b51fbcbabc7ff8fd2cfdb152421753dfeb8f7e38be9041c0bddfa1b8acd \
+                    size    2785629
+
+depends_build       port:go
```

Comment:
You can remove the `depends_build       port:go` if you are using the `golang` portgroup.